torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+tags/v2.2.0-rc8
```

Comment:
let's move this outside of `torch_patches`. I think that's what we want to do for release branches.

Reply:
Done!